README.md

```diff
@@ -39,21 +38,11 @@ Simply add it to the folder containing your component. The browser test command
 
 ## Environment
 
-We haven't yet added any utilities to help load environment variables; you can do it yourself by typing in the following in your terminal:
+This project requires several environment variables be configured before it is able to run. It uses a library called [`habitat`](https://github.com/brianloveswords/habitat) to load configuration from a `.env` file.
```

Comment:
technically it does a little more: it unifies your process and cli variables with any from-file variables, so even if you set all vars purely through the environment, habitat will allow things to "just work". Not sure how much of that deserves mention in the README, but it seemed worth pointing out